src/packages/solid/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,150 @@
+import { Component, createSignal } from 'solid-js';
+import { Show, type IntrinsicNodeProps, View, Text } from '@lightningjs/solid';
+import ProgressBar, { type ProgressBarProps } from '../ProgressBar/ProgressBar';
+import Badge, { type BadgeProps } from '../Badge/Badge';
+import Checkbox, { type CheckboxProps } from '../Checkbox/Checkbox';
+import Metadata, { type MetadataProps } from '../Metadata/Metadata';
+import styles from './Tile.styles';
+import { withPadding } from '@lightningjs/solid-primitives';
+import Label, { type LabelProps } from './Label';
+import Artwork, { type ArtworkProps } from './Artwork';
+withPadding;
+
+export interface TileProps extends TileStyleProps, IntrinsicNodeProps {
+  /**
+   * String to img source for artwork
+   */
+  artwork: Partial<ArtworkProps>;
+  /**
+   * Object containing all properties supported in the [Badge component](?path=/docs/components-badge--text)
+   */
+  badge?: Partial<BadgeProps>;
+  /**
+   * Source of logo
+   */
+  logo?: string;
+  /**
+   * Object containing all properties supported in the [Checkbox component](?path=/docs/components-checkbox--checkbox)
+   */
+  checkbox?: Partial<CheckboxProps>;
+  /**
+   * If true, changes format of itemLayout to circle
+   */
+  circle?: boolean;
+  /**
+   * Conveys the status of a program ex. "Live" or "On Now"
+   */
+  label?: Partial<LabelProps>;
+  /**
+   * Controls where there metadata is displayed in relation to the Tile. Available values are 'standard' and 'inset'
+   */
+  metadataLocation: 'standard' | 'inset';
+  /**
+   * Object containing all properties supported in the [MetadataTile component](?path=/docs/components-metadatatile--metadata-tile)<br /> Can use a different Metadata component by passing in a 'type' and then that component's properties
+   */
+  metadata?: Partial<MetadataProps> | undefined;
+  /**
+   * Metadata will be shown at all times if set to true, otherwise it will only show when the Tile has focusMetadata will be shown at all times if set to true, otherwise it will only show when the Tile has focus
+   */
+  persistentMetadata?: boolean;
+  /**
+   * Object containing all properties supported in the [ProgressBar component](?path=/docs/components-progressbar--progress-bar)
+   */
+  progressBar: Partial<ProgressBarProps>;
+  /**
+   * total width of the component
+   */
+  width: number;
+}
+
+export interface TileStyleProps {}
+
+const Tile: Component<TileProps> = (props: TileProps) => {
+  const [isFocused, setIsFocused] = createSignal(false);
+
+  return (
+    <node
```

Comment:
the answer's a bit long-winded, so bear with me - Im worried that If we were to start breaking them out into different Tiles we’ll have a whole bunch due to all the potential combinations. It can also make for a more confusing API since a user would need to keep in mind exactly which Tile does which things. That's compounded by the fact that often our users don't 100% know which things will or won't be rendered until runtime. Beyond that, we end up with a lot of duplicated code which makes maintenance a bit of a pain since we end up needing to update things in multiple places.

one option could be to create a Tile that is just the wrapper around the content, I did a little bit of exploration around this in the demo app. but that makes managing the positioning of `inset` vs `standard` metadata tricky

Another option is to create a simpler OS Tile(perhaps even the above idea). however this could put us in the territory of maintaining duplicate components across open and inner source and that’s less than ideal too. 

All of this gets into a whole conversation around how opinionated the Tile is and how it should be handled from an OS prospective